### PR TITLE
Release/TR-731 old media height fix

### PR DESCRIPTION
**Pull request by meefox**
Update @oat-sa/tao-core-ui to 1.22.5

### CODE REVIEW

File: views/package-lock.json [hunk 1]
```diff
@@ -197,7 +197,7 @@
             "resolved": "https://registry.npmjs.org/raphael/-/raphael-2.2.0.tgz",
             "integrity": "sha1-qdhPJj7I/obS4WzCz36pq8IA1ho=",
             "dependencies": {
-                "eve": "git://github.com/adobe-webplatform/eve.git#eef80ed"
+                "eve": "eve@git://github.com/adobe-webplatform/eve.git#eef80ed"
```
Comment (wazelin) [inline views/package-lock.json:200]: I'm pretty sure that has been failing the last time.

**Review by wazelin — CHANGES_REQUESTED** [review]

File: views/package-lock.json (hunk 1 above)
Comment (jsconan) [reply views/package-lock.json:200]: Indeed, this is breaking

**Review by jsconan — CHANGES_REQUESTED** [review]
The package lock is breaking the install

**Review by jsconan — CHANGES_REQUESTED** [review]
I see several issues:
- the PR is directly made against master, while it should first go to develop
- the ticket number is wrong
- thee package-lock file introduces error upon install

File: views/package-lock.json (hunk 1 above)
Comment (meefox) [reply views/package-lock.json:200]: wow I did use 7.6.3 for npm i -> update .lock file

**Review by wazelin — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Pull request closed without merging**